lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -155,7 +155,7 @@ def _gitinfo():
 def version(prog=None):
     """ return ansible version """
     if prog:
-        result = [" ".join((prog, __version__))]
+        result = ["{0} [core {1}] ".format(prog, __version__)]
```

Comment:
_Also:_ Maybe "core engine"? To me, the mention of "core" in the terminology sounds like it implies some essential underlying library that is rather internal while "engine" would be semantically closer to an app than a lib...

Reply:
I don't mind the "core" name as I think that will match the package name we're moving to.

Reply:
I would like to see this backported to 2.10 to help users understand the version drift between base^Wcore and the Ansible package -- i.e, ``ansible==2.10.4`` currently ships ``ansible-base==2.10.3`` and ``ansible --version`` reports 2.10.3.

With that in mind, maybe in 2.10 it would be called base and in >=2.11 it would be called core or maybe something entirely different. I don't know.